files/docker-volume

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+set -e
+echo -e "\nRun at $(date)"
+
+command="$1"
+host_backup_dir=/tmp/container-data-backup
+container_backup_dir=/backup
+container_name="$2"
+container_data_dirs=("${@:3}")
```

Comment:
when expecting positional parameters in your script, it might make sense to test whether they have been set and provide some helpful output if an argument is missing

Reply:
done in 081baf7